spring-cloud-commons/src/main/java/org/springframework/cloud/client/loadbalancer/CompletionContext.java

```diff
@@ -19,22 +19,46 @@
 import org.springframework.core.style.ToStringCreator;
 
 /**
+ * Allows to propagate data related to load-balanced call completion status.
```

Comment:
Change to

Allows propagation of data related to load-balanced call completion status.